### DISCUSSION TITLE
What are your plans with numpy?

### DISCUSSION THREAD

**Discussion by goodversemulti** [elimintz/justpy#207] [General]
Hi, first of all thank you for this framework. It's really amazing. I'd love to see it take off. I'd like to ask what are your plans on maintaining the framework. I see online courses have just started to cover it. Do you plan to maintain it?

**Comment by elimintz** [comment]
Wow, didn't realize that there are course covering JustPy. Could you please post a link?

Yes, I am constantly working on the framework and plan to maintain it.

**Reply by qiuwei** [reply]
@elimintz FYI, https://pythonhow.com/video-course/data-analysis/making-a-highcharts-graph/
